ios/PurchasesHybridCommon/PurchasesHybridCommon/PaywallProxy.swift

```diff
@@ -57,9 +60,20 @@ extension PaywallProxy: PaywallViewControllerDelegate {
 
     public func paywallViewController(_ controller: PaywallViewController,
                                       didFinishPurchasingWith customerInfo: CustomerInfo) {
+        self.delegate?.paywallViewController?(controller, didFinishPurchasingWith: customerInfo)
         controller.dismiss(animated: true)
     }
 
+    public func paywallViewController(_ controller: PaywallViewController,
+                                      didFinishPurchasingWith customerInfo: CustomerInfo,
+                                      transaction: StoreTransaction?) {
+        self.delegate?.paywallViewController?(controller, didFinishPurchasingWith: customerInfo, transaction: transaction)
+    }
```

Comment:
Do we need to dismiss the view controller here as well?

Reply:
Currently in native iOS, both delegate functions are called. So if we do, we would be dismissing it twice

Reply:
Honestly, we probably should only have one of these I think... But that would be part of the native SDK.

Reply:
Makes sense. Can you add a comment? If it were me, I would have added that extra dismiss 🤭

Reply:
Added in https://github.com/RevenueCat/purchases-hybrid-common/pull/586/commits/2b4526062a20e1cec7ef702618c32b6d4eb71f21

Reply:
See my other comment regarding https://github.com/RevenueCat/purchases-ios/pull/3517
So we can remove the comment too :)